plugins/drill-coverage-plugin/src/adminPartMain/kotlin/CoverageController.kt

```diff
@@ -118,7 +118,7 @@ class CoverageController(private val ws: WsService, id: String) : AdminPluginPar
         return ""
     }
 
-    private suspend fun CoverageController.calculateCoverageData(
+    suspend fun calculateCoverageData(
```

Comment:
private modificator removed due to unavaliability of private methods for debugging